pytket/extensions/qiskit/qiskit_convert.py

```diff
@@ -377,8 +383,8 @@ def add_qiskit_data(self, data: "QuantumCircuitData") -> None:
                 params = [param_to_tk(p) for p in instr.base_gate.params]
                 n_base_qubits = instr.base_gate.num_qubits
                 sub_circ = Circuit(n_base_qubits)
-                # use base gate name for the CircBox (shows in renderer)
-                sub_circ.name = instr.base_gate.name.capitalize()
+                # Use U for the name of the base gate in QControlBox
```

Comment:
Do you think its better to call the base gate U or use the original name?

I guess I was wondering if it'd be confusing if a tket circuit has a gate that looks like CS in the renderer. The CS gate isn't directly supported and the gate would actually be a QControlBox.

Reply:
I think u is fine, just do it as you like.